solutions/kubernetes/cortx-deploy.sh

```diff
@@ -138,6 +143,9 @@ case $ACTION in
         check_params
         setup_cluster cortx-cluster
     ;;
+    --collect-logs)
```

Comment:
this is log collection or generation?

Reply:
Functionality wise its generation. In ticket it said log collector script, will update to generation.